### PR TITLE
add thoughtworks

### PR DESCRIPTION
**Pull request by alicetragedy**

### CODE REVIEW

File: _data/packages.yml [hunk 1]
```diff
@@ -59,6 +59,21 @@
         Use it as a mailing list, discussion forum, long-form chat room, and more!
       years:
         - 2018
+    - name: Thoughtworks
+      id: thoughtworks
+      url: http://www.thoughtworks.com
```
Comment (carpodaster) [inline _data/packages.yml]: https://

**Review by carpodaster — APPROVED** [review]
Looks good! Just one minor issue about the protocol scheme

File: _data/packages.yml (hunk 1 above)
Comment (alicetragedy) [reply _data/packages.yml]: oh yes!! thank you 🤦‍♂️

**Comment by alicetragedy** [comment]
@carpodaster thanks for noticing! I copied this over from last year 🙈 
so I took this opportunity to update some of the links of other sponsors as well (not all had https). 🎉

**Review by carpodaster — APPROVED** [review]

**Pull request merged**